client/src/components/TrackGroup/PurchaseOrDownloadAlbumModal.tsx

```diff
@@ -76,26 +77,40 @@ const PurchaseOrDownloadAlbum: React.FC<{
                 margin-top: 0rem;
               `}
             >
+              <Button
+                variant="outlined"
+                className={css`
+                  display: block !important;
+                  ${!trackGroupId ? "display: none !important;" : ""}
```

Comment:
When would this ever be non-existent? Is this essentially saying "don't show the purchase button on the group pages?" That feels like it should be defined by a variable to the component, not defined by the URL. 

```
<PurchaseOrDownloadAlbumModal
  trackGroup={trackGroup}
  showAsLink
  />
```

And then here simply have something like 
```
css`${showAsLink ? "" : ""}` 
```

Would be a lot less code re-use

Reply:
Ah this looks good, yeah the point was to not display the button style unless : 

1/ It's trackgroup AND It's browser view